OWNERS

```diff
@@ -40,7 +40,7 @@
     },
     {
       pattern: '**/package{,-lock}.json',
-      owners: [{name: 'renovate'}],
+      owners: [{name: 'renovate[bot]'}],
```

Comment:
Since we now support bots as user owners, let's make this the same as the one in the approvers group. The effect is the same (an approving PR review from a) an owner and b) a reviewer-group member) but it's a bit simpler to look at
```suggestion
      owners: [{name: 'renovate-approve[bot]'}],
```

Reply:
Some additional context:
1. All upgrades are authored by `renovate[bot]`.
2. Some upgrades are auto-approved by `renovate-approve[bot]`.

If 1 is true but not 2, we still want a build on-duty who is a reviewer but not an owner (e.g. a subpackage upgrade) to be able to manually approve and merge a PR after CI is green.

Based on this, I'd imagine it makes sense to leave the code as is?